.github/ISSUE_TEMPLATE/Currency_update_request.yml

```diff
@@ -0,0 +1,45 @@
+name: Currency Update Request
+title: "[Currency_Update]: "
+description: "Request to fix, or update a package in the Currency Portal"
+labels: "update currency"
+projects: "ppc64le/1"
+assignees: "ICH"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Please complete the following fields to request a fix to an existing package, or an update to an existing package.
+  - type: input
+    id: pkg_name
+    attributes:
+      label: Package Name
+      description: >-
+        The package name you want to be supported on power e.g. redis, elasticsearch
```

Comment:
"power" should we say "Linux on Power" to be very specific?

Reply:
done